src/main/java/Problem1.java

```diff
@@ -2,7 +2,18 @@
 
     // Do not change signature (function name, parameters)
     public static int binarySearch(int[] data, int target) {
-        // Add your solution
+        //track the index position in the array
+        int index = 0;
+
+        //for every int element in the array
+        for(int integer : data){
+            if(target == integer){
+                return index;
```

Comment:
-10 this is linear search not binary search

Reply:
using the wrong algorithm will cost you most of the points for a problem in later homework.